content/_index.md

```diff
@@ -23,14 +27,12 @@ OSM is designed with the following core principles:
 * Integrate with external certificate management services/solutions with a pluggable interface
 * Onboard applications onto the mesh by enabling automatic sidecar injection of Envoy proxy
 
-## Install
-
-The `osm` CLI is the simplest way to install OSM on your Kubernetes cluster. To install the `osm` CLI, download and unpack the `osm` binary from the [Releases page](https://github.com/openservicemesh/osm/releases) and add it to `$PATH`. For more details on installing the `osm` CLI, see [Installation](/docs/install/).
-
-To install OSM on your cluster using the `osm` CLI, use the `install` command. For example:
+## Use Case
 
-```shell
-$ osm install
-```
+_As an operator of services spanning diverse compute platforms (Kubernetes and Virtual Machines on public and private clouds) I need an open-source solution, which will dynamically_:
 
-Alternatively, you can install OSM on your cluster using a Helm chart. You can also use the `osm` CLI to install OSM on OpenShift. For more details on installation options, see [Install OSM](/docs/install/#install-osm).
+- **Apply policies** governing TCP & HTTP access between peer services
```

Comment:
Since we support more than just TCP and HTTP, such as gRPC, and because clients do not need to be fronted by a service:
```suggestion
- **Apply policies** governing traffic access between peer applications
```